src/app/loaders/express.loader.js

```diff
@@ -1,31 +1,35 @@
-const { mapErrors } = require('../utils');
-const HttpStatus = require('http-status-codes');
 const express = require('express');
+const passport = require('passport');
+const cors = require('cors');
 const {
   userRoutes,
   groupRoutes,
-  userGroupRoutes
+  userGroupRoutes,
+  authRoutes
 } = require('../routes');
 const getLogger = require('../logger');
+const getErrorHandlerMiddleware = require('../error-handling').middlewares.errorHandler;
+const getJwtAuthMiddleware = require('../auth').middlewares.jwtAuthenticate;
+const strategies = require('../auth').strategies;
 
 async function load({ app }) {
   const logger = getLogger();
+  const jwtAuthMiddleware = getJwtAuthMiddleware(passport);
+  const errorHandlerMiddleware = getErrorHandlerMiddleware({ logger });
+  app.use(cors());
   app.use(express.json());
+  app.use(passport.initialize());
+  passport.use('jwt', strategies.jwt);
   app.use('*', (req, res, next) => {
```

Comment:
by the way it is nice to have two middleware to track request handling time. or you can just use existing solution.